db/seeds.rb

```diff
@@ -5,3 +5,20 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# Criteria and assicated records
+stakeholder_names = [ 'Local communities', 'Indiginous peoples', 'Non-governmental organisation (NGO)', 'Priivate sector (business and industry)', 'Sub-national or local government' ]
```

Comment:
'Indigenous' - this also appears in `test/fixtures/criteria.yml` (x5) and `test/fixtures/stakeholders.yml` (x2)

Reply:
And 'Private'